test/onnx/expect/TestOperators.test_linear.expect

```diff
@@ -1,43 +1,34 @@
 ir_version: 6
 producer_name: "pytorch"
-producer_version: "CURRENT_VERSION"
+producer_version: "1.8"
 graph {
   node {
     input: "input"
-    input: "weight"
+    input: "6"
+    output: "4"
+    name: "MatMul_0"
+    op_type: "MatMul"
+  }
+  node {
+    input: "4"
     input: "bias"
-    output: "3"
-    name: "Gemm_0"
-    op_type: "Gemm"
-    attribute {
-      name: "alpha"
-      f: 1
-      type: FLOAT
-    }
-    attribute {
-      name: "beta"
-      f: 1
-      type: FLOAT
-    }
-    attribute {
-      name: "transB"
-      i: 1
-      type: INT
-    }
+    output: "5"
+    name: "Add_1"
+    op_type: "Add"
   }
   name: "torch-jit-export"
   initializer {
     dims: 5
-    dims: 4
     data_type: 1
-    name: "weight"
-    raw_data: "\212\332\356>@\265u>p\303E\275 \320\306\274\354\201\221>\004\354\261\276\2746*>8\247)\276\340\035\224>\024\2446\276\200\211\312<\224\344,>D\356\257>\320\202\226\275\364\213\351>z\226\330\276\310\250\266\275\352F\377\276\000\250)=\244K\021>"
+    name: "bias"
+    raw_data: "\324BO\276@\245T>\350\377\245\275\374u\336\276&\212\304>"
   }
   initializer {
+    dims: 4
     dims: 5
     data_type: 1
-    name: "bias"
-    raw_data: "\324BO\276@\245T>\350\377\245\275\374u\336\276&\212\304>"
+    name: "6"
```

Comment:
Is there a way to keep the parameter name for `weight` (instead of `6`), like `bias` above?

Reply:
This is the traced graph:

```
graph(%self : __torch__.torch.nn.modules.linear.Linear,
      %input : Float(3, 4, strides=[4, 1], requires_grad=0, device=cpu)):
  %8 : Tensor = prim::GetAttr[name="bias"](%self)
  %7 : Tensor = prim::GetAttr[name="weight"](%self)
  %6 : Float(3, 5, strides=[5, 1], requires_grad=1, device=cpu) = aten::linear(%input, %7, %8) # /private/home/eellison/miniconda3/lib/python3.7/site-packages/torch/nn/functional.py:1753:0
  return (%6)
  ```

I think ONNX could just have a pass that sets the debugName of `prim::GetAttr` nodes to be the attribute name. What do you think ?

Reply:
after looking at the history for a bit, it looks like the renaming happens due to that constant folding kicked in for weight + transpose, since onnx::matmul is exported instead of onnx::gemm. 
For this i guess a tweak on the test case adding flag `do_constant_folding=False` when calling `_export` should work. 

I'm curious about the previous error with shuffle net. Let me take a look why it didn't work.

Reply:
The shufflenet error was before i removed the if statement on the aten::linear lowering

Reply:
Tried a few steps, that one is a legit issue on how we handle if node dynamic axes export. Could you help open a issue? You can land this one if the above suggestion works, and we can submit a fix afterwards.

Reply:
The shufflenet error has been fixed on this PR (there may still be a general issue with dynamic axes). Where do you want me to open issue ?

Reply:
I was thinking an issue summarizing the error when linear lowers to if statement of addmm and matmul. It's ok we can refer to this PR for tracking as well.

Reply:
Issue in onnxruntime or in pytorch i meant ?

Reply:
pytorch, the issue looks like in exporter, we are working on it.